src/FundraiserMailingGenerator/BaseComponent.tsx

```diff
@@ -56,7 +56,7 @@ class Generator extends Component<void, State> {
 
   async componentDidMount() {
     const rates = await fetchRates();
-    this.setState({ rates });
+    this.setState({ rates, url: 'https://actions.eko.org/a/donate' });
```

Comment:
@osahyoun I created this new release PR because the states were fetched from local storage & do not reflect the actions.eko.org update. I was not able to catch it on development because the vercel preview URLs were always new & didn't have any local storage attached. 

This is the only piece of change, I'm going ahead with the release, let me know if this PR needs any updates.